guidelines/index.html

```diff
@@ -1242,6 +1256,8 @@
 					<p>A touchscreen PDA has a keyboard interface built into its operating system as well as a connector for external keyboards. Applications on the PDA can use the interface to obtain keyboard input either from an external keyboard or from other applications that provide simulated keyboard output, such as handwriting interpreters or speech-to-text applications with "keyboard emulation" functionality.</p>
 					<p>Operation of the application (or parts of the application) through a keyboard-operated mouse emulator, such as MouseKeys, does not qualify as operation through a keyboard interface because operation of the program is through its pointing device interface, not through its keyboard interface.</p>
 				</dd>
+        <dt><dfn> known to be reliable and achievable</dfn></dt>
+        <dd>Identified in the WCAG techniques as possible to reliably correct an errors, or found in user testing to constantly</dd>
```

Comment:
"as possible to reliably correct an errors" perhaps better "as possible to reliably error-correct"

"or found in user testing to constantly" dangling sentence